.rubocop.yml

```diff
@@ -78,3 +78,4 @@ Style/AccessorMethodName:
   Exclude:
     - 'lib/rx/**/*'
     - 'lib/sm/**/*'
+    - 'lib/mhv_logging/**/*'
```

Comment:
Yes, i like using get_ and put_ for methods that are making those types of http calls and this rule prevents me from doing so.